admin/views/tabs/social/facebook.php

```diff
@@ -34,9 +34,21 @@
 	$yform->textinput( 'og_frontpage_desc', __( 'Description', 'wordpress-seo' ) );
 
 	// Offer copying of meta description.
-	echo '<input type="hidden" id="meta_description" value="', WPSEO_Options::get( 'metadesc-home-wpseo' ), '" />';
-	echo '<p class="label desc"><a href="javascript:;" onclick="wpseoCopyHomeMeta();" class="button">', esc_html__( 'Copy home meta description', 'wordpress-seo' ), '</a></p>';
+	$homepage_meta_description = WPSEO_Options::get( 'metadesc-home-wpseo' );
+	if ( ! empty( $homepage_meta_description ) ) {
+		$copy_home_meta_desc_help = new WPSEO_Admin_Help_Panel(
+			'copy-home-meda-desc',
+			__( 'Help on copying the home meta description', 'wordpress-seo' ),
```

Comment:
Could you add `esc_html` to this translation?

Reply:
https://github.com/Yoast/wordpress-seo/pull/8894/files (already merged in 7.0) adds the escaping to the button text directly in `WPSEO_Admin_Help_Panel->get_button_html()`

Reply:
As we need to revert that change (see https://github.com/Yoast/wordpress-seo-premium/issues/1598), could you add the escaping here?

Reply:
It seems that the code tags in Premium are the only occurrence of HTML in a toggle.
Will adjust that code to make sure we can escape inside this method.